### PR TITLE
Add initial buildkite pipeline for Benchmark

### PR DESCRIPTION
**Pull request by kaisecheng**
skeleton pipeline for development

### CODE REVIEW

File: .buildkite/scripts/benchmark/config/filebeat.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+- type: log
+  symlinks: true
+  paths:
+  - "/usr/share/filebeat/flog/*.log"
+logging.level: info
+output.logstash:
+  hosts:
+    - "localhost:5044"
+  compression_level: 0
+  pipelining: 3
```
Comment (jsvd) [inline .buildkite/scripts/benchmark/config/filebeat.yml:12]: interesting, why 3? I'd suggest leaving the default here.

Comment (kaisecheng) [reply .buildkite/scripts/benchmark/config/filebeat.yml]: Agree to maintain the default behavior. 
Base on my last benchmark test, 3 gave more eps.

File: .buildkite/scripts/benchmark/config/pipelines.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
+- pipeline.id: main
+  config.string: |
+    input { 
+      beats {
+        port => 5044
+        enrich => none
```
[the remark this answers is not in the document]
Comment (kaisecheng) [reply .buildkite/scripts/benchmark/config/pipelines.yml]: Default `enrich` will duplicate the size of event. But I think the size doesn't matter in this test. I mainly want to get the trend of benchmark result.
Removed the line

**Review by jsvd — APPROVED** [review]
LGTM

**Pull request merged**